scripts/validate_releases.sh

```diff
@@ -71,6 +74,9 @@ do
     xcodegen generate
     ${XCODEBUILD} -resolvePackageDependencies && ACTUAL_XCODEBUILD_RESULTS[${NAVIGATION_SDK_VERSION}]=$? || ACTUAL_XCODEBUILD_RESULTS[${NAVIGATION_SDK_VERSION}]=$?
 
+    # Validate whether Mapbox Navigation version in generated Xcode project is correct.
+    swift sh ./../../../scripts/validate_xcodeproj.swift UISPMTest.xcodeproj ${NAVIGATION_SDK_VERSION}
```

Comment:
Should we also check MapboxCoreNavigation resolved version in CoreSPMTest.xcodeproj? 
This package does not depend on Maps and it could theoretically use newer version of NavigationNative, MobileEvents and so on

Reply:
Good idea, at the same time `CoreSPMTest` works a bit differently, as it's using always current version of MapboxNavigation-SPM project. We can update it to also verify already released version (along with current one).

I'd like to land this PR as soon as possible as it'll fix filing extended workflow tests, but will create a separate ticket to implement your idea.

Reply:
We can also start extending this script to support other package managers.

Reply:
`NAVIOS-784` was created.